### ISSUE TITLE
verbose flag not working

### ISSUE THREAD

**Issue by Ancillas** [puppetlabs/r10k#32]
Running `r10k --help` prints a message that some commands are hidden, "3 hidden commands ommitted; show them with --verbose".  Running `r10k --help --verbose` results in an error, "r10k: option requires an argument -- verbose".

Passing various arguments with the --verbose flag has no affect on displaying the hidden commands.

`r10k --help --verbose true`
`r10k --help --verbose DEBUG`

**Comment by adrienthebo** [comment]
I think that at some point I noticed this issue but it didn't register long enough for me to fix it. The issue is that Cri itself was defining the `--verbose` flag and I redefined it to require an argument. Since it has to be one or the other I've changed the `--verbose` flag to take an optional level, and if there's not level then it will default to `info`. I've also modified the behavior of when `--help` is passed to make sure that it respects the `--verbose` flag, which was another  related bug.

summary: merged into master in 9669133; this should be released in 1.0.0rc4. Thanks for taking the time to report this!
